WP_Auth0.php

```diff
@@ -309,19 +309,24 @@ public function a0_register_query_vars( $qvars ) {
 	 */
 	public function wp_add_plugin_settings_link( $links ) {
 
-		$settings_link = '<a href="admin.php?page=wpa0-errors">Error Log</a>';
```

Comment:
Looks like error log page is also removed but that PR desc doesn't mention that.  Intentional?

Reply:
@damieng - Yes. I'll add to the description.